### ISSUE TITLE
Track chat command usage in stats json file

### ISSUE THREAD

**Issue by jmakhack** [jmakhack/hydrate-reminder#356]
## Feature Reasoning

For future potential features, it would be nice to track how many times the player has used every chat command.

## Feature Description

This enhancement is to increase a counter in `hydrateReminderStats.json` whenever any chat command is used. The result json should look like the following:

```json
{
    "commandCounts" : {
        "help": 5,
        "next": 2,
        "prev": 11,
        "reset": 4,
        "total": 8,
        "party": 3,
        "hydrate": 8
    },
    "totalHydrateCount": 0
}
```

Add unit tests and avoid repeating logic where possible. When new commands are added in the future, this logic should automatically be applied without any extra code.

## Possible Alternatives

Any alternatives to the proposed schema may be acceptable.

## Additional Context

Note: This work cannot be started until #350 gets merged in first.

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[<img src="https://img.shields.io/discord/1023562225790767175?color=%20%237289DA&label=discord&logo=discord&logoColor=%20%237289DA&style=for-the-badge" />](https://discord.gg/RTgxfFW9mS)

**Comment by simonkrizo** [comment]
Hello, I would like to contribute to project. Can you assign this to me?

**Comment by ron-huberfeld** [comment]
Hi @jmakhack 
Can I have this one?

**Comment by jmakhack** [comment]
Go for it @ron-huberfeld :)

**Comment by jmakhack** [comment]
@ron-huberfeld, just checking in. Are you still planning to work on this?

**Comment by ron-huberfeld** [comment]
Yes. Is it urgent. I'm a bit busy at the moment but still want to do it.

**Comment by jmakhack** [comment]
@ron-huberfeld, no worries. Just checking in on all the assigned issues :)

**Comment by ron-huberfeld** [comment]
Hi @jmakhack is this issue still open and count for hacktoberfest 2023?
